### ISSUE TITLE
Documentation has dead links

### ISSUE THREAD

**Issue by Cabalist** [Deftaudio/MacroPot#1]
Hey there,

Just building this and noticed that there are a number of places in the documentation with links that point to the incorrect places:

For instance:

pg 15: 
> Start by printing the draft label located here https://github.com/Deftaudio/MacroPot/Acrylic%20panels/MacroPot-drill_allignment.pdf, cutting it to the size and aligning to the enclosure:

should be 

> Start by printing the draft label located here https://github.com/Deftaudio/MacroPot/blob/master/Acrylic%20panels/MacroPot-drill_allignment.pdf, cutting it to the size and aligning to the enclosure:

There are similar issues on Pg7 and 9

Would a PR turning that into Markdown replacing the Google docs location be accepted?

**Comment by Deftaudio** [comment]
Thanks a lot, got all the links fixed. I wrote the guide before I published a git, so thanks again. 
I don't mind Markdown at all, it was just simpler to me to write Google Docs from mobile devices. There is also PDF version that's compiled: https://github.com/Deftaudio/MacroPot/blob/master/Assembly/MacroPot%20Assembly%20Guide.pdf
Basically, Doc version supposed to be a working doc only, while that one is ready version.
